index.html

```diff
@@ -299,16 +453,124 @@ <h1>
                         <a href="#">ADD ADDRESS</a>
                     </p>
                 </div>
+                <div>
+                    <fieldset>
+                        <legend>Delivery Type</legend>
+                        <div>
+                            <label>
+                                <input type="radio" name="d-type" value="urgent"> Urgent 1-2 days
+                            </label>
+                        </div>
+                        <div>
+                            <label>
+                                <input type="radio" name="d-type" value="normal"> Normal 5-7 days
+                            </label>
+                        </div>
+                    </fieldset>
+                </div>
+                <div>
+                    <label>
+                        <input type="checkbox"> I agree to
+                        <a href="#">Terms and Conditions</a>.
+                    </label>
+                </div>
                 <div>
                     <button class="primary">Submit</button>
                     <button>Cancel</button>
                 </div>
             </div>
         </div>
+        <div>
+            <h1>Custom variables</h1> 
```

Comment:
This is officially called  "Custom Properties", or less officially "CSS Variables" I'd stick to the official name in the header

Reply:
Can we just stick to "CSS Custom Properties" which is the official name. According to the spec:

- this is a property: `--something`
- this is a variable: `var()`